.travis.yml

```diff
@@ -1,9 +1,16 @@
 language: python
 
 python:
-  - "2.7"
-  - "3.3"
-  - "3.4"
+  - '2.7'
+  - '3.3'
+  - '3.4'
 
 script:
   - ONLINE=1 ./bau test
+
+after_success:
+  - coveralls
+
+env:
+  global:
+    secure: kyJzbnvLIGOyHE+erM7uOmpGQqW9JnCRLdhIegMfWrPDvk3gneTcvLHfJebJSuqObuL7/7NlenjvJ4MFocBhNIPS4smzyK+2mDgNZIJkPBPl84swDgr50M3u6Id4bTYPudc1BhmJXsgTBicZqZre2TAgA7p5Dye7fOI6DXERoLs=
```

Comment:
This is an environment variable encrypted using `travis encrypt`. You have to create a free account on coveralls.io, add nigelsmall/py2neo repository, then grab the repo token, and encrypt it using `travis encrypt COVERALLS_REPO_TOKEN=YourTokenHere `